plugins/ci-github-actions.yml

```diff
@@ -1,6 +1,6 @@
 id: ci-github-actions
-name: Github Actions
-description: Adds a github actions workflow file which is used to build, test and package the service.
+name: GitHub Actions
+description: Adds a Gitub Actions workflow file which is used to build, test and package the service.
```

Comment:
Hey @manulpatel coudl you change the newly introduced grammar mistake in the description: `Gitub` to `GitHub`

Reply:
Oh sorry for the blunder. Corretect it. Thanks